fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/exceptions/BadOrderByException.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.apirequest.exceptions;
+
+/**
+ * Bad having exception is an exception encountered when the having object cannot be built.
+ */
+public class BadOrderByException extends BadApiRequestException {
+
+    /**
+     * Constructor.
+     *
+     * @param message  Message of the exception
+     */
+    public BadOrderByException(String message) {
```

Comment:
I'm not too sure about your use cases, but I personally prefer just passing the value to the exception and making the message static state on the exception.

Reply:
Sure, in general, but our pattern in fili has always been to use the ErrorMessageFormat for tightly coordinated logging and error messaging on request errors.